docs/user/automatic-redirects.rst

```diff
@@ -1,56 +1,140 @@
-Automatic Redirects
-===================
+Automatic redirects and incoming links
+======================================
 
-Read the Docs supports redirecting certain URLs automatically.
-This is an overview of the set of redirects that are fully supported and will work into the future.
+Read the Docs makes it easier to create and manage incoming links by redirecting certain URLs automatically.
+This article explains the redirects that are automatically active for all Read the Docs projects and how they are useful.
+Automatic redirects on Read the Docs are supported long-term.
 
-Redirecting to a Page
+Redirecting to a page
 ---------------------
 
 You can link to a specific page and have it redirect to your default version.
 This is done with the ``/page/`` URL prefix.
 You can reach this page by going to https://docs.readthedocs.io/page/automatic-redirects.html.
```

Comment:
I think that just adds words without saying anything. The original seems clear?

Reply:
Maybe it's a personal thing. I usually find hard to parse "this" when there are many things involved since it depends on the context where it's written and can be ambiguos. In those cases, I tend to replace "this" with the exact thing I want to highlight in that sentence.

Reply:
I agree with @humitos that "this" can easily get confusing here. So I changed it. Managed to eliminate one word, so Hemingway would hopefully approve :)